earth2grid/healpix.py

```diff
@@ -262,13 +344,19 @@ def get_bilinear_regridder_to(self, lat: np.ndarray, lon: np.ndarray):
     def approximate_grid_length_meters(self):
         return approx_grid_length_meters(self._nside())
 
-    def reorder(self, order: PixelOrderT, x: torch.Tensor) -> torch.Tensor:
-        """Rorder the pixels of ``x`` to have ``order``"""
+    def _reorder_cpu(self, x: torch.Tensor, order: PixelOrderT):
```

Comment:
Could `reorder_from_cuda` and `_reorder_cpu` be in the same place? Would be nicer for future maintainability.

Reply:
The cpu and gpu reordering share almost no logic, so I split them at a higher level in the code.